src/pages/index.js

```diff
@@ -25,37 +26,115 @@ editProfileValidator.enableValidation();
 const addCardValidator = new FormValidator(config, newCardEditForm);
 addCardValidator.enableValidation();
 
+const avatarFormValidator = new FormValidator(config, avatarForm);
+avatarFormValidator.enableValidation();
+
+const removeCardValidator = new FormValidator(config, removeCardForm);
+removeCardValidator.enableValidation();
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "eeca6a67-c0e1-4af7-ac48-d61d2e5d5520",
+    "Content-Type": "application/json",
+  },
+});
+
+let cardSection;
+
+api.getInitialCards().then((cards) => {
+  cardSection = new Section(
+    {
+      items: cards,
+      renderer: (cardData) => {
+        const cardElement = getCardElement(cardData);
+        cardSection.addItem(cardElement);
+      },
+    },
+    ".cards__list"
+  );
+  cardSection.renderItems();
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**